apps/sms/views/conversation/js/conversation.js

```diff
@@ -80,6 +80,11 @@ var ConversationView = {
   showErrorInFailedEvent: '',
   previousSegment: 0,
 
+  /**
+   * Set to true after init is run.
+   */
+  _isReady: false,
```

Comment:
yeah I think always call view init and early return resolve when initialized might be another solution.